CHANGELOG.adoc

```diff
@@ -8,6 +8,22 @@ Das Format basiert auf link:https://keepachangelog.com/en/1.0.0[Keep a Changelog
 #281: Logs aus Tests in den target-Ordner schreiben,
 damit diese nicht ausversehen commited werden.
 
+## [12.41.1] - 2022-04-25
+IsTablePresent-Methode public setzen.
+
+## [12.41.0] - 2022-04-25
+
+Zweite Version von aero.minova.cas.servicenotifier. DO NOT USE.
```

Comment:
Statt FETT WARNEND `DO NOT USE`,
könnte man es auch einfach als experimentel bezeichnent.
Zudem wäre sowas wie `Experimentelle CAS-Erweiterung aero.minova.cas.servicenotifier für die Registrierung von Diensten erstellen`. nennen. So, weiss man nicht, wofür das ganze ist.